shared-data/js/__tests__/labwareDefSchemaV2.test.ts

```diff
@@ -1,21 +1,152 @@
+import 'regenerator-runtime/runtime'
 import path from 'path'
 import glob from 'glob'
 import Ajv from 'ajv'
 
 import schema from '../../labware/schemas/2.json'
-import type { LabwareDefinition2 } from '../types'
+import type { LabwareDefinition2, LabwareWell } from '../types'
 
-const definitionsGlobPath = path.join(
-  __dirname,
-  '../../labware/definitions/2/**/*.json'
-)
-
-const fixturesGlobPath = path.join(__dirname, '../../labware/fixtures/2/*.json')
+const definitionsDir = path.join(__dirname, '../../labware/definitions/2')
+const fixturesDir = path.join(__dirname, '../../labware/fixtures/2')
+const globPattern = '**/*.json'
 
 // JSON Schema defintion & setup
 const ajv = new Ajv({ allErrors: true, jsonPointers: true })
 const validate = ajv.compile(schema)
 
+const generateStandardWellNames = (
+  rowCount: number,
+  columnCount: number
+): Set<string> => {
+  function* generateWelllNames() {
```

Comment:
I think there in an extra `l`

```suggestion
  function* generateWellNames() {
```